capstone/src/main/webapp/cover.css

```diff
@@ -0,0 +1,107 @@
+/*
+ * Globals
+ */
+
+/* Links */
+a,
+a:focus,
+a:hover {
```

Comment:
Could we put these two attributes on the same line? I think this is the right convention but I'm not sure. If so, we should replace everywhere

Reply:
addressed in "address nits"